File: MiniNavigator/app/src/main/java/pl/pw/mini/mininavigator/data/LocationDate.java
package pl.pw.mini.mininavigator.data;

import java.io.Serializable;

public class LocationDate implements Serializable{
    public Location Loc;
    public Date Dat;

    @Override
    public String toString() {
        return "[" + Dat + "] " + Loc;
    }
}

File: GisService/GisService/Return/LocationDate.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using GisService.Model;

namespace GisService.Return
{
    public class LocationDate
    {
        public Interfaces.Location Loc { get; set; }
        public Date Dat { get; set; }
    }
}